src/index.html

```diff
@@ -11,12 +11,95 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="nav-item">
+            <a
+              class="link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="link"
+              data-qa="hover"
+              href="#"
+            >
+              LAPTOP & COMPUTERS
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="link"
+              href="#"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="link"
+              href="#"
+            >
+              TABLES
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="link"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links starts with the `#` symbol. This is not an issue here but it's better to have `href="#photo"` instead of `href="#"`.